fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -418,8 +418,8 @@ public FHIRPersistenceJDBCImpl(Properties configProps, IConnectionProvider cp) t
 
         try {
             checkModifiers(searchContext);
-            queryBuilder = new JDBCQueryBuilder((ParameterDAO) this.getParameterDao(),
-                                                (ResourceDAO) this.getResourceDao());
+            queryBuilder = new JDBCQueryBuilder(this.getParameterDao(),
```

Comment:
any particular reason for this change? 421/422 can now be on one line

Reply:
ugly, this is by my eclipse automatically